kitsune/search/v2/documents.py

```diff
@@ -1,12 +1,19 @@
 from django.utils import timezone
 
-from elasticsearch_dsl import field, Document as DSLDocument
+from elasticsearch_dsl import connections, field, Document as DSLDocument
 
 from kitsune.search import config
+from kitsune.search.v2.es7_utils import es7_client
 from kitsune.search.v2.fields import WikiLocaleText
 from kitsune.wiki.config import REDIRECT_HTML
 
 
+DEFAULT_ES7_CONNECTION = 'es7_default'
+
+
+connections.add_connection(DEFAULT_ES7_CONNECTION, es7_client())
```

Comment:
What about add the connection as `default` so we dont need to explicitly mention the alias all the time.

Reply:
Since we are migrating to ES7 but the default is the legacy ES lets stick with explicitly naming stuff.